static/js/components/SiteAddContent.tsx

```diff
@@ -16,28 +16,65 @@ import {
 } from "../query-configs/websites"
 import { getWebsiteDetailCursor } from "../selectors/websites"
 
-import { ConfigItem } from "../types/websites"
+import {
+  Website,
+  EditableConfigItem,
+  TopLevelConfigItem,
+  WebsiteStarterConfig
+} from "../types/websites"
 
 interface MatchParams {
   contenttype: string
+  contentsubtype: string
   name: string
 }
 
+function findEditableConfigItem(
+  config: WebsiteStarterConfig,
+  type: string,
+  subtype: string | null
+): EditableConfigItem | null {
+  const topLevelConfigItem: TopLevelConfigItem | null =
+    config.collections.find(
+      (config: TopLevelConfigItem) => config.name === type
+    ) || null
```

Comment:
Can we use `??` instead of `||` here and below? It wouldn't make a difference in functionality here but it would be more explicit